.github/workflows/deploy.yml

```
@@ -0,0 +1,79 @@
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Deploy to production
+        env:
+          SSH_PRIVATE_KEY: ${{ secrets.SSH_KEY }}
+          SERVER: ${{ secrets.SSH_SERVER }}
+          USER: ${{ secrets.SSH_USER }}
+          DEPLOY_PATH: ${{ secrets.SSH_PATH }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:4:13: Double quote to prevent globbing and word splitting

(shellcheck)